apps/www/components/Payment/Form/ApplePay.tsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from 'react'
+
+// Check if the website is running in a secure document.
+function isInSecureDocument(): boolean {
+  return (
+    typeof window !== 'undefined' && window?.location?.protocol === 'https:'
+  )
+}
+
+// Declare ApplePaySession property to prevent TypeScript errors
+declare global {
+  interface Window {
+    ApplePaySession: ApplePaySession | undefined
+  }
+}
+
+function isApplePayAvailable(): boolean {
+  if (!window?.ApplePaySession) {
```

Comment:
```suggestion
  if (!window.ApplePaySession) {
```

Reply:
Wouldn't this cause an error when a Browser has not defined ApplePaySession?